src/inverseRobot.js

```diff
@@ -7,7 +7,15 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const correctRobot = {};
+
+  for (const key in robot) {
+    if (correctRobot[robot[key]]) {
```

Comment:
You can declare const:
```
const value = robot[key];
```
And then it will be more readable:
```
if (correctRobot[value]) {
  ...
}

correctRobot[value] = key;
```
But it is just if you like